### PR TITLE
Rename ansible-pulp to pulp-installer

### PR DESCRIPTION
**Pull request by mikedep333**
[noissue]

re: 4606
Rename ansible-pulp to pulp-installer
https://pulp.plan.io/issues/6406

### CODE REVIEW

File: .travis/before_install.sh [hunk 1]
```diff
@@ -78,19 +78,19 @@ fi
 git clone https://github.com/pulp/pulp-openapi-generator.git
 if [ -n "$PULP_BINDINGS_PR_NUMBER" ]; then
   cd pulp-openapi-generator
-  git fetch origin +refs/pull/$PULP_BINDINGS_PR_NUMBER/merge
-  git checkout FETCH_HEAD
+  git fetch origin pull/$PULP_BINDINGS_PR_NUMBER/head:$PULP_BINDINGS_PR_NUMBER
```
Comment (fabricio-aguiar) [inline .travis/before_install.sh:81]: Nice!

Comment (mikedep333) [reply .travis/before_install.sh:81]: You can thank @daviddavis for updating the templated before_install.sh.j2 with this fix:
https://pulp.plan.io/issues/5990

Comment (mikedep333) [reply .travis/before_install.sh:81]: @fabricio-aguiar I'm curious, were you getting broken by PRs that were already merged?

Comment (fabricio-aguiar) [reply .travis/before_install.sh:81]: not here, but I got it in some plugins in the past

**Pull request merged**